account_asset_management/README.rst

```diff
@@ -0,0 +1,45 @@
+Financial asset management.
+===========================
+
+This Module manages the assets owned by a company. It will keep
+track of depreciation's occurred on those assets. And it allows to create
+accounting entries from the depreciation lines.
+
+The full asset life-cycle is managed (from asset creation to asset removal).
+
+Assets can be created manually as well as automatically
+(via the creation of an accounting entry on the asset account).
+
+Excel based reporting is available via the 'account_asset_management_xls'
+module (cf. http://odoo.apps.com).
```

Comment:
I'd remove the url (which is incorrect anyway), since account_asset_management_xls is in the same repo.